packages/web/src/components/search/DataSearch.js

```diff
@@ -184,6 +189,15 @@ class DataSearch extends Component {
 		this.props.removeComponent(this.internalComponent);
 	}
 
+	// returns size and aggs property
+	getBasicQueryOptions = () => {
+		let queryOptions = { size: 20 };
```

Comment:
@ShahAnuj2610 Define the default value for `size` prop as `20` & read it from props.

Reply:
@bietkul the `size` prop is for `composite` param of `aggs` => https://github.com/appbaseio/reactivecore/blob/01b710f1862dbb8101919bee5c3d6f143d4c29fb/src/utils/helper.js#L477
this `size: 20` is for the `query` param

Reply:
@ShahAnuj2610 It should support both.